### PR TITLE
[MGDOBR-194] Increased access token lifespan to 24hrs in local dev and demo environments

### PR DESCRIPTION
**Pull request by robpblake**
This commit adds two changes:
- Made the access token for the API last for 24hrs in local dev and our demo environment
- Ensured we are consistent across local dev and demo environment in the version of Keycloak that we are using

[MGDOBR-194](https://issues.redhat.com/browse/MGDOBR-194) 

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: kustomize/overlays/prod/manager/keycloak-realm.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
...
...hUov4OZ77LMLX2uJTFTg2S7YuDsyKcT4p5nsmvtn3/zY58tpl0w8s3QT2PNbhyPrb64+msgg0TY0bDxbIYnW2T9QQFM6fh0CGvA+Gav5QL6uvCzrByxjZeDzybh16w0DoG/R93EVOsrjtRrgFTVlPDUbAeaXVJy6YPNOE6vMZyLCkoa27A5NdFpKNYmwiqeZRDgDCkOIfya+DFNNHKi6eRg7AyWt2Na6o4lCECsEh+GID0G3Ugz1G+UucXlnMi9/8mp8UFLRp+d4OtfWa61rHV0MGe7fXBW7twV9J1qSvWbS4S7GFQqilPdtG0PZhZWjeaHe9bACkKQkrK059+L+q3BpllXR2h7Ye05JTyQPTFXso2u9b4Z7DouxQ/a/nHkCBYT4e66Mbg6Yuc/13u80B3VPetJngH3h9qen/inSxQUDRryrl2TUwiuwbneP2DHmuWLp6yIrs0IeGZppLOzd3GvsrC31G8shD+lEVxtgPX2dE0C6PrzeTOpai8b+1f1oPLW+bhRBkpkI+d8DnJAMcEeBM6EXTPXrF1XMdFZiuakbDnvOdTTmonJGnK1PA6sX0Tjp2Q/YohQtVi1oZsqa9/6xYbV4Kr6+xM1jur64LSlLG8wtwlZhOiNnsatEbrAkryBe1AUULiWiGipL5rqWQ9OjO6YwK2E3PHaCWZYu659+4r3iCIIaPMukCrA0W3CkkSvjL2x677tFNpjIix0n+NPJBfJAzshlUSDE7vA6GsEtfLsHhm5mxNHjpBb7tbP4kduyvC6dP9eMdLzp1au62Q4T2JUp4XReTRrqPhtBWa/YzRE7U7TRiXBY7VOR1RDyTDskFf7wkwOQ+1jlG8xJKiiTiMBFj8rFWR6vLBHhqMuOuIXe9jJH9qcgwSliGofXp04fU2tTU0VLtw6iW4k+Pvc/zUvhwuQO/zelnLIYQdgOb70AjUmX+WZ/0gJ5ARUyMsJueJ1Rp2tOM0AarXxD7B7lZXEiiy/xhcBFlUuC85mkmsDdaUgLdhza3umJMMpBD9rJLW02ixEz03ZWX9ZWOIKyiPdGxAn+EpYwV1LytrJTd8bzRo+XDvSOUaXhfe+KcBJ+A/FvDQYI5MpHr7bD3Od+u/vjF7ELAUf8rX3TvJgLal7p2j5zNCPNMj6XeonDML7SQdQIGHEgj2c8AyFfi0zUPp7c/mvgXGlC9cjtHur3DQ==
```
Comment (robpblake) [inline kustomize/overlays/prod/manager/keycloak-realm.yaml:10]: I have ensured that our demo environment credentials are the same as before, but this now includes the extended realm configuration from the Keycloak export.

**Review by r00ta — COMMENTED** [review]
Hey @robpblake , I'd suggest to only add the following line 
```
      "attributes": {
        "access.token.lifespan": "86400"
      }
```
to the `event-bridge` client. The new config file is huge (and I think not necessary), it's pretty hard to make changes to it. At the end the config file should look like this 

```json
{
  "realm": "event-bridge-fm",
  "displayName": "event-bridge",
  "enabled": true,
  "sslRequired": "external",
  "registrationAllowed": false,
  "requiredCredentials": [ "password" ],
  "users": [
    {
      "username": "admin",
      "enabled": true,
      "credentials": [
        {
          "type": "password",
          "value": "123"
        }
      ],
      "applicationRoles": {
        "realm-management": [
          "manage-users",
          "manage-clients"
        ],
        "account": [
          "manage-account"
        ]
      }
    },
    {
      "username": "kermit",
      "enabled": true,
      "credentials": [
        {
          "type": "password",
          "value": "thefrog"
        }
      ],
      "applicationRoles": {
        "cloud-services": [
          "user"
        ],
        "event-bridge": [
          "user"
        ]
      }
    }
  ],
  "roles": {
    "realm": [],
    "client": {
      "event-bridge": [
        {
          "name": "user",
          "composite": false,
          "clientRole": true,
          "containerId": "event-bridge-fm"
        }
      ],
      "cloud-services": [
        {
          "name": "user",
          "composite": false,
          "clientRole": true,
          "containerId": "cloud-services"
        }
      ]
    }
  },
  "defaultRoles": [],
  "requiredCredentials": [
    "password"
  ],
  "scopeMappings": [],
  "clients": [
    {
      "id": "e1bc1111-69ba-4bf2-b342-189e1c2a98ef",
      "clientId": "event-bridge",
      "surrogateAuthRequired": false,
      "enabled": true,
      "standardFlowEnabled": true,
      "alwaysDisplayInConsole": false,
      "directAccessGrantsEnabled": true,
      "publicClient": true,
      "clientAuthenticatorType": "client-secret",
      "redirectUris": [
        "*"
      ],
      "webOrigins": [
        "*"
      ],
      "bearerOnly": false,
      "protocol": "openid-connect",
      "attributes": {
        "access.token.lifespan": "86400"
      }
    },
    {
      "id": "e6b8dc94-8dbb-40e8-a32c-fbc352dce8fb",
      "clientId": "cloud-services",
      "enabled": true,
      "alwaysDisplayInConsole": false,
      "directAccessGrantsEnabled": true,
      "fullScopeAllowed": true,
      "clientAuthenticatorType": "client-secret",
      "publicClient": true,
      "redirectUris": [
        "*"
      ],
      "webOrigins": [
        "*"
      ],
      "bearerOnly": true
    }
  ],
  "keycloakVersion": "15.1.0"
}
```

**Comment by robpblake** [comment]
> Hey @robpblake , I'd suggest to only add the following line
> 
> ```
>       "attributes": {
>         "access.token.lifespan": "86400"
>       }
> ```
> 
> to the `event-bridge` client. The new config file is huge (and I think not necessary), it's pretty hard to make changes to it. At the end the config file should look like this
> 
> ```json
> {
>   "realm": "event-bridge-fm",
>   "displayName": "event-bridge",
>   "enabled": true,
>   "sslRequired": "external",
>   "registrationAllowed": false,
>   "requiredCredentials": [ "password" ],
>   "users": [
>     {
>       "username": "admin",
>       "enabled": true,
>       "credentials": [
>         {
>           "type": "password",
>           "value": "123"
>         }
>       ],
>       "applicationRoles": {
>         "realm-management": [
>           "manage-users",
>           "manage-clients"
>         ],
>         "account": [
>           "manage-account"
>         ]
>       }
>     },
>     {
>       "username": "kermit",
>       "enabled": true,
>       "credentials": [
>         {
>           "type": "password",
>           "value": "thefrog"
>         }
>       ],
>       "applicationRoles": {
>         "cloud-services": [
>           "user"
>         ],
>         "event-bridge": [
>           "user"
>         ]
>       }
>     }
>   ],
>   "roles": {
>     "realm": [],
>     "client": {
>       "event-bridge": [
>         {
>           "name": "user",
>           "composite": false,
>           "clientRole": true,
>           "containerId": "event-bridge-fm"
>         }
>       ],
>       "cloud-services": [
>         {
>           "name": "user",
>           "composite": false,
>           "clientRole": true,
>           "containerId": "cloud-services"
>         }
>       ]
>     }
>   },
>   "defaultRoles": [],
>   "requiredCredentials": [
>     "password"
>   ],
>   "scopeMappings": [],
>   "clients": [
>     {
>       "id": "e1bc1111-69ba-4bf2-b342-189e1c2a98ef",
>       "clientId": "event-bridge",
>       "surrogateAuthRequired": false,
>       "enabled": true,
>       "standardFlowEnabled": true,
>       "alwaysDisplayInConsole": false,
>       "directAccessGrantsEnabled": true,
>       "publicClient": true,
>       "clientAuthenticatorType": "client-secret",
>       "redirectUris": [
>         "*"
>       ],
>       "webOrigins": [
>         "*"
>       ],
>       "bearerOnly": false,
>       "protocol": "openid-connect",
>       "attributes": {
>         "access.token.lifespan": "86400"
>       }
>     },
>     {
>       "id": "e6b8dc94-8dbb-40e8-a32c-fbc352dce8fb",
>       "clientId": "cloud-services",
>       "enabled": true,
>       "alwaysDisplayInConsole": false,
>       "directAccessGrantsEnabled": true,
>       "fullScopeAllowed": true,
>       "clientAuthenticatorType": "client-secret",
>       "publicClient": true,
>       "redirectUris": [
>         "*"
>       ],
>       "webOrigins": [
>         "*"
>       ],
>       "bearerOnly": true
>     }
>   ],
>   "keycloakVersion": "15.1.0"
> }
> ```

@r00ta I did all changes via the UI for Keycloak, so the workflow is 

- Make changes in the UX
- Export the file
- Commit it

Feels simple enough to me. Do we need to edit the file by hand?

**Comment by r00ta** [comment]
@robpblake with the minimal config file I posted it's possible to have a look at the current configuration of keycloak and keep it under control: it's easy to understand what are the realms, the users etc... a 100 lines config file is much better than a 2k one. 
With the entire dump of the configuration this is not possible (or at least, we lose control of what we really configured).

**Review by r00ta — APPROVED** [review]

**Pull request merged**